app/components/candidate_interface/application_status_tag_component.html.erb

```diff
@@ -22,22 +22,13 @@
       classes: 'govuk-!-margin-bottom-0 govuk-!-margin-top-2',
     ) do %>
       <p>
-        Some providers allow you to defer your offer. This means that you could start your course a year later.
+        You can defer your offer which means that you could start your course a year later.
       </p>
 
       <p>
-        Every provider is different, so it may or may not be possible to do this.
-        Find out by contacting <%= @application_choice.current_course_option.course.provider.name %>.
+        Contact <%= @application_choice.current_course_option.course.provider.name %> to ask if it’s possible to defer, this will not affect your existing offer. If your provider agrees, you’ll need to accept the offer on your account first and then defer it.
       </p>
 
-      <p>
-        Asking if it’s possible to defer will not affect your existing offer.
-      </p>
-
-      <% if @application_choice.offer? %>
-        <p>
-          If your provider agrees to defer your offer, you’ll need to accept the offer on your account first.
-        </p>
       <% end %>
```

Comment:
i think the problem is that you've deleted the `if` but not the corresponding `end`. If you remove line 32 (`<% end %>`), order should be restored to the universe.